src/main/java/carbonconfiglib/gui/config/ElementList.java

```diff
@@ -193,14 +193,14 @@ protected void renderBackground(PoseStack stack) {
 		if(customBackground == null || (minecraft.level != null && customBackground.shouldDisableInLevel())) return;
 		renderBackground(x0, x1, y0, y1, (float)getScrollAmount(), customBackground.getTexture());
 	}
-	
+
 	@Override
-	protected void renderList(PoseStack stack, int mouseX, int mouseY, float partialTicks) {
-		super.renderList(stack, mouseX, mouseY, partialTicks);
+	protected void renderList(PoseStack poseStack, int i, int j, int k, int l, float f) {
```

Comment:
The first two variables after poseStack is left and top.
Please properly name variables in the style it was before.